### PR TITLE
csi-kata-directvolume: add Dockerfile for building csi image

### PR DESCRIPTION
**Pull request by Apokleos**
Fixes: #9163

### CODE REVIEW

File: src/tools/csi-kata-directvolume/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2024 Ant Group
+
+FROM alpine
+LABEL maintainers="Kata Containers Authors"
+LABEL description="Kata DirectVolume Driver"
+ARG binary=./bin/directvolplugin
+
+# Add util-linux to get a new version of losetup.
+RUN apk add util-linux coreutils e2fsprogs xfsprogs xfsprogs-extra btrfs-progs && apk update && apk upgrade
```
Comment (justxuewei) [inline src/tools/csi-kata-directvolume/Dockerfile:9]: Please split this into three lines to enhance readability.

```suggestion
RUN apk add util-linux coreutils e2fsprogs xfsprogs xfsprogs-extra btrfs-progs \
    && apk update \
    && apk upgrade
```

Comment (Apokleos) [reply src/tools/csi-kata-directvolume/Dockerfile]: Thx @justxuewei fixed

File: src/tools/csi-kata-directvolume/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2024 Ant Group
+
```
Comment (Tim-Zhang) [inline src/tools/csi-kata-directvolume/Dockerfile:2]: SPDX-License-Identifier is missing here

Comment (Apokleos) [reply src/tools/csi-kata-directvolume/Dockerfile:4]: Thx @Tim-Zhang fixed

**Review by Tim-Zhang — APPROVED** [review]

File: src/tools/csi-kata-directvolume/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+FROM alpine:3.19
+
+LABEL maintainers="Kata Containers Authors"
+LABEL description="Kata DirectVolume Driver"
+ARG binary=./bin/directvolplugin
+
+RUN apk add --no-cache util-linux=2.39.3-r0 coreutils=9.4-r2 e2fsprogs=1.47.0-r5 xfsprogs=6.5.0-r0 xfsprogs-extra=6.5.0-r0 btrfs-progs=6.6.2-r0 \
+    && apk update \
+    && apk upgrade \
+    && rm -rf /var/cache/apk/*
```
Comment (justxuewei) [inline src/tools/csi-kata-directvolume/Dockerfile:14]: This line is unnecessary since `--no-cache` is already provided.

Comment (Apokleos) [reply src/tools/csi-kata-directvolume/Dockerfile]: Thx, fixed

**Review by justxuewei — COMMENTED** [review]
Lgtm, thanks!

**Review by justxuewei — APPROVED** [review]

**Pull request merged**